### ISSUE TITLE
Master doesn't compile because of a typo

### ISSUE THREAD

**Issue by real-felix** [raydium-io/raydium-library#8]
```
error[E0609]: no field `with_matedata` on type `raydium_amm_v3::instruction::OpenPositionV2`
   --> /Users/felixdv/.cargo/git/checkouts/raydium-library-35e3e5b78a9011c5/51bb799/libraries/clmm_cli/src/decode_clmm_ix_event.rs:315:46
    |
315 |                         with_metadata: instr.with_matedata,
    |                                              ^^^^^^^^^^^^^ unknown field
    |
help: a field with a similar name exists
    |
315 |                         with_metadata: instr.with_metadata,
    |                                              ~~~~~~~~~~~~~
```

Hey, I suggest you to look at what a CI is, it would help with this kind of stuff.

**Comment by real-felix** [comment]
#9

**Comment by RainRaydium** [comment]
OK, thanks, we have fixed in the commit db25e7c9a5db9312c15807b3a50e7ea855578633
